File: app/src/main/java/com/arad/care4pets/HealthRecordsActivity.java
package com.arad.care4pets;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;

public class HealthRecordsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_health_records);

        Button btnAddHealthRecord = findViewById(R.id.fabAddHealthRecord);
        ImageButton btnBack = findViewById(R.id.btnBack);

        btnAddHealthRecord.setOnClickListener(v -> {
        Intent intent = new Intent(HealthRecordsActivity.this, AddHealthRecordActivity.class);
        startActivity(intent);
    });

        btnBack.setOnClickListener(v -> finish());
    }
}

File: app/src/main/java/com/arad/care4pets/HealthRecordsActivity.kt
package com.arad.care4pets

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity

class HealthRecordsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_health_records)

        val btnAddHealthRecord = findViewById<Button>(R.id.btnAddHealthRecord)
        val btnBack = findViewById<ImageButton>(R.id.btnBack)

        btnAddHealthRecord.setOnClickListener {
            startActivity(Intent(this, AddHealthRecordActivity::class.java))
        }

        btnBack.setOnClickListener {
            finish()
        }
    }
}
